modules/aws-outposts-load-balancer-clb.adoc

```diff
@@ -0,0 +1,160 @@
+
+[NOTE]
+====
+If you do not need to prevent a Classic Load Balancer in the VPC cluster from targetting Outpost edge compute nodes, you do not need to complete these steps.
+====
+
+.Prerequisites
+
+* You have extended an AWS VPC cluster into an Outpost.
+
+* You have access to the cluster using an account with `cluster-admin` permissions.
+
+* You have installed the {oc-first}.
+
+.Procedure
+
+. Optional: Verify that the edge compute nodes have the `location=outposts` label by running the following command and verifying that the output includes only the edge compute nodes in your Outpost:
++
+[source,terminal]
+----
+$ oc get nodes -l location=outposts
+----
+
+. Label the cloud-based compute nodes in the VPC cluster with a key-value pair by running the following command:
++
+[source,terminal]
+----
+$ for NODE in $(oc get node -l node-role.kubernetes.io/worker --no-headers | grep -v outposts | awk '{print$1}'); do oc label node $NODE <key_name>=<value>; done
+----
++
+where `<key_name>=<value>` is the label you want to use to distinguish cloud-based compute nodes.
++
+.Example output
+[source,text]
+----
+node1.example.com labeled
+node2.example.com labeled
+node3.example.com labeled
+----
+
+. Optional: Verify that the cloud-based compute nodes have the specified label by running the following command and verifying that the output includes all cloud-based compute nodes in your VPC cluster:
++
+[source,terminal]
+----
+$ oc get nodes -l <key_name>=<value>
+----
++
+.Example output
+[source,terminal]
+----
+NAME                   STATUS    ROLES     AGE       VERSION
+node1.example.com      Ready     worker    7h        v1.28.5
+node2.example.com      Ready     worker    7h        v1.28.5
+node3.example.com      Ready     worker    7h        v1.28.5
+----
+
+//Might be able to remove the deployment part and refer to user workloads as a pre-req.
+. Configure the Classic Load Balancer `deployment` resource:
++
+[source,yaml]
+----
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: <application_name>
+  namespace: <application_namespace>
+spec:
+  selector:
+    matchLabels:
+      app: <application_name>
+  replicas: 3
+  template:
+    metadata:
+      labels:
+        app: <application_name>
+    spec:
+      containers:
+        - image: openshift/origin-node
+          command:
+           - "/bin/socat"
+          args:
+            - TCP4-LISTEN:8080,reuseaddr,fork
+            - EXEC:'/bin/bash -c \"printf \\\"HTTP/1.0 200 OK\r\n\r\n\\\"; sed -e \\\"/^\r/q\\\"\"'
+          imagePullPolicy: Always
+          name: <application_name>
+          ports:
+            - containerPort: 8080
+----
+
+. Create the `deployment` resource CR by running the following command:
++
+[source,terminal]
+----
+$ oc create -f <clb_deployment>.yaml
+----
```

Comment:
Marco proposed dropping this and adding "you have created a user workload in the Outpost" as a pre-req